.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Bug Report
+about: Help us fix something that's broken.
+title: "PackageName:  General Summary of the Bug"
+labels: "type: bug"
+assignees: ''
+
+---
+
+## Bug Report
+@trilinos/PackageName
```

Comment:
Should we make this something obviously invalid, like `@trilinos/<PackageName>`?

Reply:
Made the switch.